src/main/java/org/truffleruby/core/rope/Rope.java

```diff
@@ -95,6 +95,7 @@ public final int depth() {
     @TruffleBoundary
     public int hashCode() {
         if (!isHashCodeCalculated()) {
+            // FIXME CS 2-Jan-18 RopeOperations.hashForRange uses Hashing.rawHash which isn't randomised per-process!
```

Comment:
Method renamed? I think the comment means `Hashing.stringHash`, not `Hashing.rawHash`.

Reply:
Fixed - I'll expand on this in another PR.